packages/components/ContextualMenu/src/ContextualMenuItem.tsx

```diff
@@ -62,7 +67,29 @@ export const ContextualMenuItem = compose<ContextualMenuItemType>({
       icon: !!icon
     };
 
-    const cmRef = useViewCommandFocus(userProps.componentRef);
+    const onKeyUp = React.useCallback(
```

Comment:
We might have a useKeyCallback hook you could leverage already?  Worth a look.

Reply:
useKeyCallback seems useful for keyboard events with only one key type. I have to support both ' ' and 'Enter' so it might be just convenient for me to define this as is

Reply:
I refactored it in my PR to make it more general, we can touch it up later though.